api.go

```diff
@@ -389,6 +392,16 @@ func (server *ApiServer) Version(response http.ResponseWriter, request *http.Req
 	}
 }
 
+func (server *ApiServer) Dashboard(response http.ResponseWriter, request *http.Request) {
+	var dashboardData []proxyToxics
```

Comment:
I made a decision early on to not use the existing HTTP API for these since we already have a couple of functions  handy for fetching them.